mlir/test/lib/Dialect/Tosa/TosaTestPasses.cpp

```diff
@@ -165,18 +165,20 @@ ConvertTosaConv2DOp::matchAndRewrite(Operation *op,
   // Obtain the quantized scale = multiplier and shift.
   computeMultiplierAndShift(opTensorScale, multiplier, shift, 32);
 
-  bool input_unsigned =
-      newTosaConv2DOp.getResult().getType().isUnsignedInteger();
-  bool output_unsigned = outputType.isUnsignedInteger();
+  bool inputUnsigned = inputType.getElementType().isUnsignedInteger();
```

Comment:
I think this is the inputType for the conv2d operation, would the input to the rescale be `outputType`?

Reply:
Done

Reply:
nit: I think we can just simplify to
```
bool input_output_unsigned = outputType.isUnsignedInteger();
```
(apologies this is what I was referring to in the previous review 😅)